src/main/scala/com/gu/anghammarad/Anghammarad.scala

```diff
@@ -33,10 +33,10 @@ object Anghammarad {
     }
   }
 
-  def send(contact: Contact, message: Message): Try[Unit] = {
+  def send(contact: Contact, message: Message): Try[SendEmailResult] = {
```

Comment:
I've changed this stuff a bit in that other PR and elsewhere, sorry. Let's do some work to merge what you have into the latest version.